jormungandr/src/settings/start/config.rs

```diff
@@ -123,6 +123,14 @@ pub struct P2pConfig {
     #[serde(default)]
     pub gossip_interval: Option<Duration>,
 
+    /// if no gossip has been received in the last interval, try to connect
+    /// to nodes that were put in quarantine and for which quarantine has
+    /// elapsed.
+    ///
+    /// The default value is 5 min.
+    #[serde(default)]
+    pub network_stuck_check: Option<Duration>,
```

Comment:
Document this setting here, please. https://github.com/input-output-hk/jormungandr/blob/master/doc/configuration/network.md

Actually, this may need a larder update since it is already outdated and I am also thinking is it is possible to generate a piece of that file based on the structure documentation. Probably, using something like [`cargo-sync-readme`](https://crates.io/crates/cargo-sync-readme).

Reply:
Good catch!
Part of the outdated published doc is because our doc actions has been failing for at least 2 months